### PR TITLE
removing base-component and fixing lint errors

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers here's a little cleanup pr

### CODE REVIEW

File: react-native/react/engine/index.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ class Engine {
     this.generalListeners[method] = (this.generalListeners[method] || []).filter(l => l !== listener)
   }
 
-  _rpcWrte (data) {
+  _rpcWrite (data) {
```
Comment (chrisnojima) [inline react-native/react/engine/index.js]: typo

File: react-native/react/search/index.desktop.js [hunk 2]
```diff
@@ -17,7 +15,7 @@ export default class Search extends BaseComponent {
   }
 
   buildRows (results) {
-    console.log("results: ", results)
+    console.log('esults: ', results)
```
Comment (cjb) [inline react-native/react/search/index.desktop.js]: typo

**Comment by cjb** [comment]
:+1: %C

File: react-native/react/engine/index.js (hunk 1 above)
Comment (MarcoPolo) [inline react-native/react/engine/index.js]: I'm so sorry, here's a :cookie:

File: react-native/react/search/index.desktop.js (hunk 2 above)
Comment (chrisnojima) [inline react-native/react/search/index.desktop.js]: image

**Comment by Sidnicious** [comment]
Looks legit, :shipit:.

**Pull request merged**